include/zephyr/net/ethernet.h

```diff
@@ -715,6 +715,22 @@ static inline bool net_eth_is_addr_lldp_multicast(struct net_eth_addr *addr)
 	return false;
```

Comment:
Typo in the commit msg s/except/expect/

Reply:
Also the commit subject should start with `net: ethernet: ...` as this is Ethernet specific change and not really related to generic l2 layer.

Reply:
> Typo in the commit msg s/except/expect/
Thank you @jukkar I have changed the relevant commit message

In wiki: https://en.wikipedia.org/wiki/Precision_Time_Protocol
Multicast group address table:  All except peer delay messages

Reply:
Ok, my bad, I thought it was a typo.